src/HomePage/HomePage.js

```diff
@@ -1,29 +1,39 @@
-import React from 'react';
-import { View, Image } from 'react-native';
-import { Button } from "react-native-paper";
-import { getAllRooms } from '../../apiCalls';
-import Overview from '../Overview/Overview'
+import React, { Component } from "react";
+import { View, Image, TouchableOpacity } from "react-native";
+import { getAllRooms } from "../apiCalls";
 
-const HomePage = () => {
-  const handlePress = () => {
+class HomePage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rooms: [],
+    };
+  }
+  resolvedRooms = async () => {
+    await getAllRooms(1)
+      .then((data) => this.setState({ rooms: data.data }))
+      .then((response) =>
+        this.props.navigation.navigate("Overview", { rooms: this.state.rooms })
```

Comment:
Is the second argument for the "navigate" method the props that are being passed to Overview?